src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,47 @@
+package mate.academy.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import java.time.LocalDateTime;
+
+@Entity
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    private Movie movie;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    private CinemaHall cinemaHall;
+
+    private LocalDateTime showTime;
+
+    public void setShowTime(LocalDateTime showTime) {
```

Comment:
Consider adding getter methods for the properties of this entity. Accessing entity fields directly is not a recommended practice as it breaks encapsulation. Providing getters allows for better control over the class's properties.